app/Http/Controllers/PresetsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class PresetsController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return string
+     */
+    public function index()
+    {
+        // All these events take 1 day and repeat yearly
+        $yearly = json_decode('[
```

Comment:
These will have to be fetched from an Open Data source, or generated through a library (preferably an Open Data Source). But this is fine for now.